lib/taskbook.js

```diff
@@ -307,13 +307,16 @@ class Taskbook {
   checkTasks(ids) {
     ids = this._validateIDs(ids);
     const {_data} = this;
+    const [checked, unChecked] = [[], []];
 
     ids.forEach(id => {
       _data[id].isComplete = !_data[id].isComplete;
+      return _data[id].isComplete ? unChecked.push(id) : checked.push(id);
     });
 
     this._save(_data);
-    render.markComplete(ids);
+    render.markIncomplete(checked);
+    render.markComplete(unChecked);
```

Comment:
Checked items should be displayed before the unchecked ones, just like with the display of the starred/unstarred items on line 476:

```js
 render.markComplete(checked);
 render.markIncomplete(unchecked);
```